panel/styles/models/html.less

```diff
@@ -0,0 +1,34 @@
+.copybtn {
+  position: absolute;
```

Comment:
I would expect the copy button to have a color aligned with the (dark) theme. But it has not.

https://github.com/user-attachments/assets/f5de2143-3aa5-4e2a-b159-228d33c97529

```python
import panel as pn
from panel.chat import ChatInterface

pn.extension("perspective")

code = """
Some code

```python
a=1
b=a+1
print(a, b)
```
"""

pn.template.FastListTemplate(
    title="Copy Button",
    sidebar=[code],
    main=[code],
).servable()
```

Reply:
There could be inspiration here:

- https://docs.streamlit.io/develop/api-reference/text/st.code

Reply:
This is how chatgpt looks for me. Could also be inspiration.

https://github.com/user-attachments/assets/3df18770-5a9a-4300-89f5-03e02771c3b7